app/src/main/java/io/euphoria/xkcd/app/impl/ui/UIUtils.java

```diff
@@ -196,6 +197,16 @@ private static String normalize(String text) {
         return EMOJI_RE.matcher(text).replaceAll("").replaceAll("[^\\w_\\-]", "").toLowerCase();
     }
 
+    /**
+     * Trim all unicode whitespace characters from the start and end of the passed String.
+     *
+     * String::trim() uses a non-unicode-aware (or only partially so) concept of whitespace characters.
+     */
+    @NonNull
+    public static String trimUnicodeWhitespace(@NonNull String text) {
+        return text.replaceAll("^\\p{Z}+|\\p{Z}+$", "");
```

Comment:
For consistency with other places (and to avoid redundant recompilations), it is preferrable to use a `private static final Pattern` instead.

Reply:
resolved by 8fd33c557c91207155b0e8ee559109b4a749e25e